_posts/2016-11-07-recurse-center-week-6.md

```diff
@@ -43,10 +43,10 @@ I had a lot of bottled up praise that I wanted to share. I had even considered p
 
 It gave me a sense of closure that I could share those thoughts with all the outgoing RCers.
 
-I was surprised at my feelings after sharing those things. Letting others know my feelings is important to me, and I expect others to value receiving that information.
+I was surprised by how I felt after sharing those things. Letting others know my feelings is important to me, and I expect others to value receiving that information.
 
-Some people thanked me afterwards for my niceities and that felt really good. Knowing that they valued hearing my thoughts as much as I valued sharing them.
-Other people didn't say anything. As if whatever I had to say was just another "It was nice to meet you!". That bothered me a bit.
+Some people thanked me afterwards for my niceities and that felt really good... knowing that they valued hearing my thoughts as much as I valued sharing them.
+Other people didn't acknowledge it. It was as if whatever I had to say was just another "It was nice to meet you!". That surprised me a bit.
```

Comment:
Oh yay! I think of all the things I wrote about this week, it was the hardest thing to admit. 

I don't want to come across as passive aggressive "Oh, I see how it is", but at the same time I like to notice where feelings are reciprocated so that I can invest my time in those places. And notice where it isn't so I can know who: 

a) Aren't also interested in investing time.
b) Don't have compatible communication styles.

Glad you appreciated it!